src/polymer/polymer-element.ts

```diff
@@ -60,6 +60,10 @@ export interface Options {
     javascriptNode: estree.Expression | estree.SpreadElement,
     expression: LiteralValue
   }[];
+  listeners?: {
+    event: string,
+    handler: string
```

Comment:
I was thinking of adding autocompletion of listener values. E.g. `foo: '|'` autocompleting to all available methods in the current element.

For that I assume `getLocationInfoForPosition` needs to know the sourceRange of the block, to know whether the user currently wants to autcomplete listeners.

The comment is not a blocker, just something I was thinking of implementing.